tests/key_clef_tests.ts

```diff
@@ -22,18 +23,12 @@ const ClefKeySignatureTests = {
   },
 };
 
-function getWidth(code: string) {
-  const el = new Element();
-  el.setText(code);
-  return el.getWidth();
-}
-
 const fontWidths = () => {
-  const sharpWidth = getWidth('\ue262' /*accidentalSharp*/) + 1;
-  const flatWidth = getWidth('\ue260' /*accidentalFlat*/) + 1;
+  const sharpWidth = Element.getElementWidth(Glyphs.accidentalSharp) + 1;
```

Comment:
There is already a `Element.getWidth()`. It is unclear how this is different (if I were a developer new to VexFlow).

Should this be called `Element.measureWidth()`?

Reply:
done